packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -901,18 +913,80 @@ export class JobService {
   }
 
   @Cron(CronExpression.EVERY_10_MINUTES)
-  public async launchCronJob() {
-    this.logger.log('Launch jobs START');
+  public async createEscrowCronJob() {
+    const isCronJobRunning = await this.cronJobService.isCronJobRunning(
+      CronJobType.CreateEscrow,
+    );
+
+    if (isCronJobRunning) {
```

Comment:
This might be moved into isCronJobRunning

Reply:
Sorry, I'm not following.

Reply:
I am talking about this line this.logger.log('Previous cron job is not completed yet');